ipn/backend.go

```diff
@@ -122,11 +122,11 @@ type Notify struct {
 	// is available.
 	ClientVersion *tailcfg.ClientVersion `json:",omitempty"`
 
-	// Full set of current TailfsShares that we're publishing as name->path.
+	// Full set of current TailFSShares that we're publishing as name->path.
```

Comment:
// TailFSShares is ...

(complete sentence starting with the subject matching the field/var/type/etc name)

Reply:
Fixed